contracts/src/v0.8/shared/test/call/CallWithExactGas.t.sol

```diff
@@ -216,6 +220,41 @@ contract CallWithExactGas__callWithExactGasSafeReturnData is CallWithExactGasSet
     assertGt(gasUsed, 500);
   }
 
+  function test_CallWithExactGasSafeReturnData_ConsumeAllGas_Success() external {
+    uint16 maxRetBytes = 0;
+    uint256 miscGasOverhead = 128;
```

Comment:
pls document how the overhead is calculated as 128; also it should be a constant right?